tests/data/api/api-visual-barchart.json

```diff
@@ -34,6 +34,137 @@
   "url": "/summary/visual/assessment/2/barchart/",
   "url_update": "/summary/visual/assessment/2/barchart/update/",
   "url_delete": "/summary/visual/assessment/2/barchart/delete/",
+  "rob_settings": {
```

Comment:
`rob_settings` is shown at the root visual level and it looks like it may be repeated at the the study level for each study - should we use a different study serializer that doesn't include `rob_settings`?

Reply:
Well... as @rabstejnek indicated this was a little more involved than I expected. 

Keeping as is even though it's sending excess data b/c it allows us to used the cached Study objects, so the additional db hits are small.

We should try to remove these in the future. I added a task to describe what would be required. The Study javascript object currently unpacks risk of bias as well. We should refactor the Study object so that the study just returns study info and there are separate api requests for rob data. Then, the visual could just get the simple study, and the final rob for a list of studies.